basc_archiver/sites/base.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, handler_callback, options):
             raise Exception('BaseSiteArchiver must be subclassed!')
         self.options = options
         self.base_thread_dir = os.path.join(options.base_dir,
-                                            '{}/{{board}}/{{thread}}/'.format(self.name))
+                                            '{{board}}/thread/{{thread}}/'.format())
```

Comment:
`self.name` needs to be subbed in as the first folder to separate out different sites, ie 4chan vs 8chan vs etc